.projenrc.ts

```diff
@@ -64,6 +64,7 @@ const project = new awscdk.AwsCdkConstructLibrary({
     'typedoc-plugin-markdown',
     'aws-sdk-mock',
     '@aws-cdk/assert',
+    '@aws-cdk/integ-tests-alpha',
```

Comment:
should we add the explicit version here, to make sure it matches the cdk version ? CDK_VERSION-alpha.0

Reply:
good catch! Yes, will update it accordingly

Reply:
just updated the code to include this with the latest commit